backend/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunDAO.java

```diff
@@ -826,6 +835,16 @@ public FutureExperimentRunDAO(
         executor);
   }
 
+  private boolean checkAllResourceAllowed(List<Resources> resources) {
+    boolean allowedAllResources = false;
+    for (Resources resources1 : resources) {
+      if (resources1.getService().equals(ServiceEnum.Service.MODELDB_SERVICE)) {
```

Comment:
This should always be the case unless we have a bug.

Reply:
Fixed